### DISCUSSION TITLE
Sound notification

### DISCUSSION THREAD

**Discussion by LfJohnVo** [munafio/chatify#201] [Q&A] [answered]
How i can add a sound notification?

**Comment by munafio** [comment, marked as answer]
@LfJohnVo Hi, you can implement it easily! 

an `mp3` file for your sound included in the DOM and can be played only when a message received event triggered.

**Reply by Evraldouvci** [reply]
the received message event already exists?

**Reply by munafio** [reply]
@Evraldouvci yes! how can we receive messages real-time if it's not exists?
Follow [this](https://github.com/munafio/chatify/blob/f20fa3fedd4067a36adb0a0fd225d72e61aa9e91/src/assets/js/code.js#L614) commented line, this method/listener responsible for listening on messages coming.

**Reply by munafio** [reply]
you can put the logic there, I mean you can play a notification sound there when receiving a message

**Comment by munafio** [comment]
Now v1.6.0 will support notifications sound
